Client/src/models/game/availableActions/AvailableActions.java

```diff
@@ -265,7 +268,23 @@ public Boolean canReplace(CompressedPlayer player) {
         if (player.getPlayerNumber() != GameController.getInstance().getCurrentGame().getCurrentTurnPlayer().getPlayerNumber()) {
             return false;
         }
+        return getNumTimesReplacedThisTurn() < getMaxNumReplacePerTurn();
```

Comment:
I'm confused by this return statement.    Why would a 'targetInRange' function need to know how many times I've replaced ??

I see the comment, which is something I think I may have wrote.  So maybe the problem is that this function needs a better name?

Basically, if I can't tell what this function does just by looking at it, then that means the code is unclear. The good news is this can probably be fixed with better naming/comments.

Reply:
Renaming the variables will be in a different commit, gotta keep things agile